### ISSUE TITLE
Not entirely open source?

### ISSUE THREAD

**Issue by Anonymous2716** [Hax4us/Apkmod#160]
It says uses custop apktool. But no sources available. In termux I have still issues with the apktool in the repo it can't recompile the apk. However Apkmod can I wanna know what modifications apktool has. Also it uses aapt from this repo. I'm interested in the aapt too.

**Comment by dedshit** [comment]
> It says uses custop apktool. But no sources available. In termux I have still issues with the apktool in the repo it can't recompile the apk. However Apkmod can I wanna know what modifications apktool has. Also it uses aapt from this repo. I'm interested in the aapt too.

just use original apktool do it manually

**Comment by Anonymous2716** [comment]
@dedshit  I need the aapt .

**Comment by dedshit** [comment]
> @dedshit  I need the aapt . 

search in google git

**Comment by Anonymous2716** [comment]
@dedshit  yep but this aapt seems custom. And works fine with alpine Linux arm64 .
